### PR TITLE
Followup to #1395: Additions to nodejs image file

### PR DESCRIPTION
**Pull request by bfallonf**
Followup to #1395 . Just some basic sentence/markup stuff.

@adellape @ahardin-rh @tnguyen-rh @tpoitras Take a look.

### CODE REVIEW

**Comment by bfallonf** [comment]
[rev_history]
|link:../../using_images/s2i_images/nodejs.html[Node.js]
|Added to the link:../../using_images/s2i_images/nodejs.html#configuration[Configuration] section, and included a Hot Deploying section.
%

**Comment by ahardin-rh** [comment]
@bfallonf LGTM!  :metal:

File: using_images/s2i_images/nodejs.adoc [hunk 1]
```diff
@@ -81,37 +83,37 @@ section] of the build configuration's `*sourceStrategy*` definition.
...
-For example, see the
-link:../../dev_guide/new_app.html#specifying-environment-variables[`oc new-app`]
-command. You can use the
-link:../../dev_guide/environment_variables.html#set-environment-variables[`oc
-env`] command to update environment variables of existing objects.
+You can set new environment variables when
+link:../../dev_guide/new_app.html#specifying-environment-variables[creating new
+applications], or
+link:../../dev_guide/environment_variables.html#set-environment-variables[update
+environment variables for existing objects].
```
Comment (tnguyen-rh) [inline using_images/s2i_images/nodejs.adoc]: Suggest: s/update/updating/ for parallelism w/ "creating"

**Comment by tnguyen-rh** [comment]
@bfallonf One nit, otherwise LGTM.

**Pull request merged**